apps/passport-client/src/validateState.ts

```diff
@@ -1,7 +1,7 @@
 import { User, requestLogToServer } from "@pcd/passport-interface";
 import { PCDCollection } from "@pcd/pcd-collection";
 import {
-  IdentityV4,
+  IdentityV3,
```

Comment:
Minor note: IdentityV3 duplicates the Identity definition imported below.  Probably best to remove the second import and use IdentifyV3 everywhere.

Reply:
fixed.